Mathlib/Algebra/Category/Grp/Limits.lean

```diff
@@ -86,7 +84,13 @@ noncomputable instance Forget₂.createsLimit :
   letI : (forget₂ Grp.{u} MonCat.{u}).ReflectsIsomorphisms :=
     CategoryTheory.reflectsIsomorphisms_forget₂ _ _
   createsLimitOfReflectsIso (K := F) (F := (forget₂ Grp.{u} MonCat.{u}))
-    fun c' t =>
+    fun c' t => by
+      have : Small.{u} (Functor.sections ((F ⋙ forget₂ Grp MonCat) ⋙ forget MonCat)) := by
+        have : HasLimit (F ⋙ forget₂ Grp MonCat) := HasLimit.mk {cone := c', isLimit := t}
+        apply Concrete.small_sections_of_hasLimit (F ⋙ forget₂ Grp MonCat)
+      have : Small.{u} (Functor.sections (F ⋙ forget Grp)) := inferInstanceAs <| Small.{u}
+        (Functor.sections ((F ⋙ forget₂ Grp MonCat) ⋙ forget MonCat))
+      exact
```

Comment:
`exact` is unnecessary here if you remove the `by` at the `fun c' t => by` line.
```suggestion
```
The `{ liftedCone := ... }` block should be indented by two more spaces.

Reply:
Done and done.